### DISCUSSION TITLE
What type of mining the teaproject does?

### DISCUSSION THREAD

**Discussion by puchungotv** [tearust/teaproject#6] [Q&A] [answered]
Thank you Kevin for your help.

**Comment by kevingzhang** [comment, marked as answer]
The miner will get TEA token.
TEA token is the fuel / gas of TEA network ecosystem.
TEA is used to pay for computing on layer 2 trusted nodes. The node owners (miners) receive TEA because the work it did to either a client or a public service.

The miner nodes do not need to run a hash function or proof of space-time. It needs to run RA (Remote Attestation) algorithm to verify other TEA nodes are valid. This is one of the so-called public services. Because RA algorithm doesn't require to burn energy or hard drive, but it does need a special security chip called TPM (in current version)
